src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindIntegrationTest.java

```diff
@@ -929,6 +929,37 @@ public void withSizeOperatorNoMatch() {
           .body("data.documents", hasSize(0));
     }
 
+    @Test
+    public void withSizeOperatorNotZero() {
+      String json =
+          """
+                  {
+                      "find": {
+                          "filter": {
+                              "$not": {
```

Comment:
The $not syntax that Mongoose supports is `tags: { $not: { $size: 0 } }`, not `{ $not: { tags: { $size: 0 } } }`. Does the `tags: { $not: { $size: 0 } }` syntax work here?